### DISCUSSION TITLE
Init gpu memory fail.

### DISCUSSION THREAD

**Discussion by jiecaix** [AMReX-Codes/amrex#4041] [Q&A] [answered]
When i try to use warpx in gpu, it shows that "amrex::Abort::2::Out of gpu memory. Free: 23827316736 Asked: 25567297536 !!!". Is there any way to avoid it? I try to use multi gpu (4 V100) to run this code but still need the same size of gpu memory.

**Comment by jiecaix** [comment]
A part of GPU memory has been used by other code.

**Comment by zingale** [comment, marked as answer]
you could try:
```
amrex.the_arena_init_size = 0
```

see: https://amrex-codes.github.io/amrex/docs_html/GPU.html?highlight=the_arena_init_size
